Javascript/keys.js

```diff
@@ -1,5 +1,3 @@
-const wikiEndpoint='http://en.wikipedia.org/w/api.php?origin=*&action=opensearch&format=json&search='
-
```

Comment:
I need this URL to display "suggestions".

Reply:
I am going to fix all those endpoints at some point.  Having the queries fixed instead of set as params is awful.  You are welcome to do what you will with the endpoints in the meantime, but I plan to erase the ones with the params baked into the endpoint and set that all correctly.